package/src/renderer/processors/Rects.ts

```diff
@@ -1,24 +1,18 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import type { SkRect, SkRRect } from "../../skia";
+import { Skia } from "../../skia";
 
 import { vec } from "./math/Vector";
 import type { Radius } from "./Radius";
 import { processRadius } from "./Radius";
 
-export const point = (x: number, y: number) => ({ x, y });
+export const point = (x: number, y: number) => Skia.Point(x, y);
 
-export const rect = (x: number, y: number, width: number, height: number) => ({
-  x,
-  y,
-  width,
-  height,
-});
+export const rect = (x: number, y: number, width: number, height: number) =>
```

Comment:
Changes to Point and Rect should be a part of a separate PR?

Reply:
😮 my apologies I didn't realize this was part of this PR.